ecosystem/typescript/sdk/src/aptos_account.ts

```diff
@@ -30,6 +32,34 @@ export class AptosAccount {
     return new AptosAccount(HexString.ensure(obj.privateKeyHex).toUint8Array(), obj.address);
   }
 
+  static isValidPath = (path: string): boolean => {
+    if (
+      // eslint-disable-next-line prefer-regex-literals
+      !new RegExp("^m\\/44'\\/637'\\/[0-9]+'\\/[0-9]+'\\/[0-9]+'+$").test(
```

Comment:
You can either:
`new RegExp("^m/44'/637'/[0-9]+'/[0-9]+'/[0-9]+'+$").test` Or `/^m\/44'\/637'\/[0-9]+'\/[0-9]+'\/[0-9]+'+$/.test`

Reply:
The `\\` is not needed

Reply:
I'll fix it